### PR TITLE
Added and filled in the new metadata template.

### PR DESCRIPTION
**Pull request by KarlaRFuentes**

### CODE REVIEW

File: docassemble/AppealsMotionToReconsider/data/questions/Motion_to_Reconsider.yml [hunk 1]
```diff
@@ -10,6 +10,45 @@ metadata:
...
+  estimated_completion_minutes: 60
+  estimated_completion_delta: 30
+  languages:
+    - en
+  help_page_url: https://www.mass.gov/rules-of-appellate-procedure/appellate-procedure-rule-27-motion-for-reconsideration-or-modification-of-decision
+  help_page_title: Appellate Procedure Rule 27 Motion for reconsideration or modification of decision
+  LIST_topics: 
+    - FA-07-00-00-00
+  tags: 
+    - FA-07-00-00-00
```
Comment (samglover) [inline docassemble/AppealsMotionToReconsider/data/questions/Motion_to_Reconsider.yml:40]: This tag is for [Domestic Violence and Abuse](https://taxonomy.legal/terms/FA-07-00-00-00/domestic-violence-and-abuse). I don't think there is a LIST tag directly for this, but perhaps using both of these tags would do the trick:

```
  - CR-08-01-01-00
  - CO-07-02-00-00
```

What do you think, @nonprofittechy?

File: docassemble/AppealsMotionToReconsider/data/questions/Motion_to_Reconsider.yml [hunk 2]
```diff
@@ -10,6 +10,45 @@ metadata:
...
+  jurisdiction: NAM-US-US+MA
+  original_form: 
+  original_form_published_on: 
+  review_date: August 2024
+  form_titles:
+    - Appeals Motion for Reconsideration or Modification of Decision
+  form_numbers:
+    - 
+  fees:
+    - Filing fee: 0.00
```
Comment (samglover) [inline docassemble/AppealsMotionToReconsider/data/questions/Motion_to_Reconsider.yml:50]: From [this page](https://www.mass.gov/info-details/appeals-court-fees#appeals-court-fees-) it looks like there is a filing fee for a motion to a single justice, but I don't see anything that might apply to a motion to reconsider. @nonprofittechy does that sound right—there is no filing fee for a motion to reconsider?

Comment (nonprofittechy) [reply docassemble/AppealsMotionToReconsider/data/questions/Motion_to_Reconsider.yml:50]: Right, no filing fee for the motion to reconsider

File: docassemble/AppealsMotionToReconsider/data/questions/Motion_to_Reconsider.yml (hunk 1 above)
Comment (samglover) [reply docassemble/AppealsMotionToReconsider/data/questions/Motion_to_Reconsider.yml:40]: @KarlaRFuentes With this change, I think it's done!

**Pull request merged**